buildenv/jenkins/testJobTemplate

```diff
@@ -225,9 +225,12 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							stringParam('OPENJDK_SHA', "", "OpenJDK SHA from which to run tests")
 							stringParam('TKG_OWNER_BRANCH', "AdoptOpenJDK:master", "TestKitGen git [owner]:[branch]. Default is AdoptOpenJDK:master")
 							stringParam('TKG_SHA', "", "TKG sha")
+              stringParam('ADOPTOPENJDK_SYSTEMTEST_OWNER_BRANCH', "Personal Adoptopenjdk/openjdk-systemtest repository owner:branch", "AdoptOpenJDK:master")
+              stringParam('OPENJ9_SYSTEMTEST_OWNER_BRANCH', "Personal eclipse/openj9-systemtest repository owner:branch", "eclipse:master")
+              stringParam('STF_OWNER_BRANCH', "Personal stf repo owner:branch", "AdoptOpenJDK:master")
```

Comment:
The order is wrong. It should be stringParam(parameter, value, description). For example:

```
stringParam('ADOPTOPENJDK_SYSTEMTEST_OWNER_BRANCH', "AdoptOpenJDK:master", "Personal Adoptopenjdk/openjdk-systemtest repository owner:branch")
```

Please also see the existing code in the file for examples.

Reply:
Please fix the indentation.